### ISSUE TITLE
Can not complete the authen on the https://keriganc.com/vector-epod-setup/html/main.html

### ISSUE THREAD

**Issue by oxenearth55** [kercre123/wire-prod-pod#22]
Can not complete the authen on the https://keriganc.com/vector-epod-setup/html/main.html 
When I tried to click the Activate button, it displayed
"Error logging in. Please try again. (This is normal for wire-pod at the moment, it may take a few tries to authenticate)"

**Comment by kercre123** [comment]
Your Vector is probably having trouble communicating with your wire-prod-pod instance. May you provide more info about your setup, such as what device wire-pod is running on and its operating system? And are you sure this device and your Vector are on the same network?

**Comment by oxenearth55** [comment]
Yes, I connect the vector and Labtop (Window 11) with the same network. I try to follow the step 10 which command
 **cd ~
curl -o wsl-firewall.ps1 https://keriganc.com/wsl-firewall.ps1
.\wsl-firewall.ps1** 

As a result, the powershell outputs :
 **/usr/bin/bash: line 1: ifconfig: command not found
The Script Exited, the ip address of WSL 2 cannot be found**

**Comment by oxenearth55** [comment]
Now, I can fix above error but when i click the activate button again, now it showed **Error logging in. The bot is likely unable to communicate with your wire-pod instance. Make sure you followed all of the steps and try again.**
The wire pod still be running at the port 8080

**Comment by kercre123** [comment]
Did you follow step 7 and run `sudo apt install net-tools` in the Linux terminal? I do now realize that the Powershell script uses an outdated command which may not exist anymore... I will update the script when I get around to it. Note that Windows 10/11 supoort is still experimental.

**Comment by oxenearth55** [comment]
Yes, I have done step7.

**Comment by BillMerryman** [comment]
My issue (#37 Vector won't activate) is the same. Should I delete my issue and communicate in this thread?

**Comment by BillMerryman** [comment]
I was finally able to get my Vector to activate. I think it may have had something to do with my bluetooth. My new desktop machine doesn't have integrated bluetooth. I had a dongle on it, but it was really old and wouldn't even see Vector. I have a laptop with integrated bluetooth, but it is also pretty old. It would see, connect, and appeared to download the firmware, but wouldn't activate. I went and bought a new bluetooth dongle for my desktop, supports the newest standard and backwards compatible. Hooked it up, redownloaded the firmware (because I had reverted to DDL firmware in the meantime), and got to the activate step. Clicked 'Activate' and it activated on the first try. So try making sure your bluetooth supports the newest standards. I'm using a Asus USB-BT500.
